python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -76,15 +86,37 @@ class ParquetFormatter(FormatterV2):
     """
 
     default_extension = ".parq"
+    can_read_from_uri = True
     can_read_from_local_file = True
 
     def can_accept(self, in_memory_dataset: Any) -> bool:
         # Docstring inherited.
         return _checkArrowCompatibleType(in_memory_dataset) is not None
 
+    def read_from_uri(self, uri: ResourcePath, component: str | None = None, expected_size: int = -1) -> Any:
+        # Docstring inherited from Formatter.read.
+        try:
+            fs, path = uri.to_fsspec()
+        except ImportError:
+            log.debug("fsspec not available; falling back to file access.")
```

Comment:
```suggestion
            log.debug("fsspec not available; falling back to local file access.")
```
I'm pondering how we can convince the test to run the fallback path. I assume you tested that it does work.

Reply:
So it is my understanding that the docker build does not have fsspec or s3fs, and I didn't update that build action.  So I think that does test the fallback path, even if coverage doesn't know.  Also, after I made these changes I pushed to get the GHA actions to pass before I added fsspec to the build yaml.  (But that was a one-off obviously).  I'm not sure how to regularly test both code paths without a lot of additional complications that don't seem warranted.

Reply:
Yes. That's fine. Good point about the docker tests passing without fsspec.